### PR TITLE
Avatar badges Added 

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
[Badge live demo]( https://vigilant-turing-b08150.netlify.app/badges/badges )
[Avatar live demo](https://vigilant-turing-b08150.netlify.app/avatar/avatar)

### CODE REVIEW

File: Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    text-align: center;
+    margin-top: 1rem;
+    margin-left: 1rem;
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  iframe{
+    max-width: 450px; 
+    height: 223px;
```
Comment (sanjay-xdr) [inline Avatar/avatar.css]: use relative units

Comment (sanjay-xdr) [reply Avatar/avatar.css]: used flex many times try not to repeat same code again make a separate class for flex

**Review by sanjay-xdr — COMMENTED** [review]
everything else looks good to me just use relative unit rather than px

File: Avatar/avatar.css (hunk 1 above)
Comment (syed-shaida-hussain) [reply Avatar/avatar.css]: @sanjay-xdr done! Thank you so much.

**Comment by ivinayakg** [comment]
looks good to me, go ahead and merge it.

**Review by ivinayakg — APPROVED** [review]
looks good to me, go ahead and merge it.

**Pull request merged**